internal/cmd/scraper/scraper.go

```diff
@@ -163,16 +166,44 @@ func Run(cfg *Config) {
 					err,
 				)
 			}
+
+			transportOption, err := integration.TelemetryHarvesterWithInfraTransport(
+				cfg.LicenseKey,
+				cfg.EmitterProxy,
+				cfg.EmitterCaFile,
+				cfg.EmitterInsecureSkipVerify,
+			)
+
+			if err != nil {
+				logrus.Fatalf(
```

Comment:
I'm not a fan of exiting the application outside of the main() function. Could we return an error from the `run` function and quit outside of it?